examples/containerattached/cluster.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: containerattached.gcp.upbound.io/v1beta1
+kind: Cluster
+metadata:
+  annotations:
+    meta.upbound.io/example-id: containerattached/v1beta1/cluster
+  labels:
+    testing.upbound.io/example-name: example-attached-cluster
+  name: example-attached-cluster
+spec:
+  forProvider:
+    description: Test cluster
+    distribution: aks
+    fleet:
+    - project: projects/my-project-number
+    location: us-west1
+    oidcConfig:
+    - issuerUrl: https://oidc.issuer.url
+    platformVersion: 1.23.0-gke.2
+    project: my-project-id
+
+---
```

Comment:
Could you please remove these redundant dashes? It causes the failure of publish-artifacts job.